src/main/java/core/basesyntax/FruitBatch.java

```diff
@@ -0,0 +1,45 @@
+package core.basesyntax;
+
+import java.time.LocalDate;
+import java.util.Objects;
+
+public final class FruitBatch {
+    private final String fruitType;
```

Comment:
should this field be final?

Reply:
I use FruitBatch as keys in a map in FruitStorage, so I tried to make this class immutable.